assignments/objects.js

```diff
@@ -18,27 +18,78 @@ const example = {
 }
 
 // Write your intern objects here:
+const Mitzi  = {
```

Comment:
This was intended to be in one array - I will mention in the staff channel to reword it.
Now that you do know haha, how would you switch this?